### PR TITLE
Cleanup/lun 2739

### PR DESCRIPTION
**Pull request by zoliszeredi**

### CODE REVIEW

File: cms_blogger/templates/admin/cms_blogger/blogentrypage/poster_image.html [hunk 1]
```diff
@@ -3,7 +3,10 @@
 <div class="poster-image-container">
     <div class="left-side pull-left">
         <label class="no-margin-bottom">Default Image</label>
-        <div class="help-block no-margin-top"> Image must be 16:9 aspect ratio</div>
+        <div class="help-block no-margin-top">
+	  <p> Image must be 16:9 aspect ratio. </p>
+	  <p> The recomended image size is 1280 x 720. </p>
```
Comment (alexandrupirjol) [inline cms_blogger/templates/admin/cms_blogger/blogentrypage/poster_image.html]: Can we pull this value from settings?
eg: 'The recomended image size is {} x {}.'.format(settings.POSTER_IMAGE_WIDTH, settings.POSTER_IMAGE_WIDTH % settings.POSTER_IMAGE_ASPECT_RATIO)

Comment (zoliszeredi) [inline cms_blogger/templates/admin/cms_blogger/blogentrypage/poster_image.html]: Sure we can, I'll add the change

Comment (virtosubogdan) [inline cms_blogger/templates/admin/cms_blogger/blogentrypage/poster_image.html]: small side note: this was discussed already when we wanted to add the USE_BOOTSTRAP settings flag in templates and the decision was that it is to risky to import settings in any way in templates/filters.
I think it was fixed by adding only the required settings in the context.

Comment (zoliszeredi) [inline cms_blogger/templates/admin/cms_blogger/blogentrypage/poster_image.html]: We'll add the image size dimensions in the context or the rendering function. We already expose some variables.

File: cms_blogger/utils.py [hunk 2]
```diff
@@ -88,57 +89,73 @@ def paginate_queryset(queryset, page, max_per_page):
...
+        message_format = 'Cannot open image {image}. Error occured: {error}'
+        message = message_format.format(image=image_file.name, error=e)
+        raise IOError(message)
+    image.load()
+    poster_size = calculate_resized_poster_size(image.size, specs)
+    poster_width, poster_height = poster_size
+    image.thumbnail(poster_size, resizer.ANTIALIAS)
+    thumbnail_width, thumbnail_height = image.size
+    poster_image = resizer.new('RGBA', poster_size, 'white')
+    center_point = ((poster_width - thumbnail_width) / 2,
```
Comment (virtosubogdan) [inline cms_blogger/utils.py]: We should rename this variable as it is the top left margin limit where the image must be pasted.

**Pull request merged**